solidity/contracts/deposit/Deposit.sol

```diff
@@ -282,6 +282,25 @@ contract Deposit is DepositFactoryAuthority {
         return true;
     }
 
+    /// @notice Requests a funder abort for a failed-funding deposit; that is,
+    ///         requests the return of a sent UTXO to _abortOutputScript. It
+    ///         imposes no requirements on the signing group. Signers may or may
+    ///         not send their UTXO to the requested output script, at their
+    ///         discretion and with no penalty for failing to do so. This can be
```

Comment:
I think it's worth explaining that while "there's no protocol incentive" a signer still "should" return funds, or similar? Just being clear that while the protocol can't enforce it, that doesn't mean depositors need to expect signers will steal their funds. Most signers at launch will be known entities after all

Reply:
Did a pass at this, have a look.